### DISCUSSION TITLE
Updating raspotify

### DISCUSSION THREAD

**Discussion by SCGB** [dtcooper/raspotify#586] [Q&A] [answered]
Hi, this is a real noob question and probably has to do with my poor understanding of raspotify and the RPi in general, but how do I update raspotify and anything associated (librespot?) to make sure that I have the latest development? Do I simply follow the installation procedure again? Will it overwrite the raspotify conf file and asound.conf? I have googled extensively but perhaps not using the right phrases - or, it is a non issue for reasons that I don't understand.

**Comment by JasonLG1979** [comment, marked as answer]
If you installed Raspotify via in install script Raspotify's apt repository will have been added so keeping Raspotify up to date is as simple as just keeping your system up to date as a whole. You can look up `apt` Debian's package manager for full details and instructions.

**Reply by SCGB** [reply]
Excellent, thanks! I found and ran the sudo apt update and sudo apt upgrade commands. There was a ton of updates so probably I have never done this before.

A follow-up question though: I got the question whether I wanted to update or keep the raspotify conf file. I was worried that there might be stuff in it that I wouldn't remember how to reconfigure, so I kept it. Any updates to that file that I might be missing? I did the original install around 8 months ago.

**Reply by JasonLG1979** [reply]
The wiki has the current default config for reference and you can always make a backup and compare them.
